### PR TITLE
started redo of admin landing page

### PR DESCRIPTION
**Pull request by emmaeagles**
going to make a second branch for this i think bc the next part is going to be a big boy i think

# Summary

added buttons to admin homepage to serve as quick links to frequent areas of co-operator 

## Test Plan

How did you test your changes?

i tried it :-) screenshots will be commentd below

## Related Issues

Which issues does this PR resolve/work on?
#103 but not done yet

### CODE REVIEW

File: Frontend/src/boot/common.js [hunk 1]
```diff
@@ -14,6 +14,21 @@ const common = {
     }
     return yrs;
   },
+  getCurrentYear: function() {
+    return new Date().getFullYear();
+  },
+  getCurrentTerm: function() {
+    var date = new Date();
+    var month = date.getMonth();
+    if (month == 0 || month == 1 || month == 2 || month == 3) {
```
Comment (kraglalbert) [inline Frontend/src/boot/common.js]: Lol just do `if (month <= 3)`

File: Frontend/src/boot/common.js [hunk 2]
```diff
@@ -14,6 +14,21 @@ const common = {
...
+      return "WINTER";
+    } else if (month == 4 || month == 5 || month == 6 || month == 7) {
```
Comment (kraglalbert) [inline Frontend/src/boot/common.js]: Do a similar thing for the other conditions

File: Frontend/src/pages/admin/AdminCoopReviewPage.vue [hunk 3]
```diff
@@ -70,6 +70,13 @@ import CoopReviewPageNewCoopItem from "../../components/admin/CoopReviewPageNewC
 
 export default {
   name: "AdminCoopReviewPage",
+  props: {
+    currentTab_p: {
```
Comment (kraglalbert) [inline Frontend/src/pages/admin/AdminCoopReviewPage.vue]: I would change this to `currentTab` and change the data one to `currentTabData`

File: Frontend/src/pages/admin/AdminStudentsPage.vue [hunk 4]
```diff
@@ -46,15 +46,37 @@
...
+    status_p: {
+      type: String,
+      required: false,
+      default: ""
+    },
+    term_p: {
+      type: String,
+      required: false,
+      default: ""
+    }
```
Comment (kraglalbert) [inline Frontend/src/pages/admin/AdminStudentsPage.vue]: Same comment here for the prop variable names

File: Frontend/src/router/routes.js [hunk 5]
```diff
@@ -55,14 +55,20 @@ const routes = [
...
+        path: "coops",
+        component: () => import("pages/admin/AdminCoops.vue")
+      },
+      {
+        name: "Review",
         path: "coops/review",
         component: () => import("pages/admin/AdminCoopReviewPage.vue")
       },
       {
+        name: "loadofshit",
```
Comment (kraglalbert) [inline Frontend/src/router/routes.js]: LOL change this before actually merging it in

**Comment by emmaeagles** [comment]
image

**Comment by kraglalbert** [comment]
@emmaeagles you don't have to do it in this PR necessarily but I would group the buttons in some way, like maybe make a "Filter by Course" section and put the FACC200, FACC201, etc. buttons under there

File: Frontend/src/pages/admin/AdminStudentsPage.vue [hunk 6]
```diff
@@ -46,17 +46,39 @@
...
     statusOptions: [],
-    status: "",
-    seasons: [],
-    season: "",
+    status_data: "",
+    terms: [],
+    term_date: "",
     years: [],
-    year: "",
+    year_data: "",
```
Comment (kraglalbert) [inline Frontend/src/pages/admin/AdminStudentsPage.vue]: nit: use camelCase for variable names

**Pull request merged**